cosmos-sdk-proto/Cargo.toml

```diff
@@ -19,6 +19,9 @@ rust-version = "1.72"
 prost = "0.12.3"
 prost-types = "0.12"
 tendermint-proto = "0.37"
+serde = "1.0.203"
+pbjson = "0.6.0"
+pbjson-types = "0.6.0"
```

Comment:
These don't support [`Any::from_msg`](https://docs.rs/prost-types/latest/prost_types/struct.Any.html#method.from_msg) and [`Any::to_msg`](https://docs.rs/prost-types/latest/prost_types/struct.Any.html#method.to_msg) which are used quite a bit in downstream code (at least mine) beyond this repo.

Per [this discussion](https://github.com/cosmos/cosmos-rust/pull/459/files/3d574b0fbced3f4dee9fdcee9dd338efacc99d4c#diff-2566f86e46e2addbeabb507d6df353b42a4017362e4626c7c501f77d0aa27439) the best way forward here might be to generate our own `google.protobuf` types and use those

Reply:
Here's an issue tracking that. We can hopefully get `Any::{from_msg, to_msg}` and `pbjson` both supported in tendermint-rs itself: https://github.com/informalsystems/tendermint-rs/issues/1445

Reply:
#482 switches to using the new `tendermint-rs` definitions for these types which support the `no_std`-friendly fork of `pbjson`